### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by gugelius**
First homework is ready for check!

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        String userName = "";
+        while (userName.isEmpty()){
+            userName = scanner.nextLine().trim();
+            if(userName.isEmpty()){
+                System.out.println(CORRECT_VALUE_MSG);
+            }
+        }
+
+        while (!backToMainMenu) {
+            System.out.println("User Menu:");
```
Comment (elnurmes) [inline src/Main.java]: try to use java text blocks: https://www.baeldung.com/java-text-blocks

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            System.out.println("1. Browse available spaces");
+            System.out.println("2. Make a reservation");
+            System.out.println("3. View my reservations");
+            System.out.println("4. Cancel a reservation");
+            System.out.println("5. Back to main menu");
+            try {
+                switch (scanner.nextInt()) {
+                    case 1 -> manager.printAvailableSpaces();
+                    case 2 -> {
+                        if (manager.isSpacesEmpty()) {
```
Comment (elnurmes) [inline src/Main.java]: create separate method and use it here

Comment (elnurmes) [reply src/Main.java]: same for other switches

**Review by elnurmes — CHANGES_REQUESTED** [review]

**Comment by elnurmes** [comment]
@gugelius Hi! In general good job) I left some comments, pls fix them

**Comment by gugelius** [comment]
Solved problems you mentioned before, ready for check!

**Review by elnurmes — APPROVED** [review]
Hi! you can merge it

**Pull request merged**